prep `2.14.18` (#11769)
PrefectHQ/prefect: RELEASE-NOTES.md

# Prefect Release Notes

## Release 2.14.18

### Fixes

> - Fix environment variable not accepting lists — https://github.com/PrefectHQ/prefect/pull/11722

Reviewer: This should maybe just be "allow prefect settings to accept lists?" i think?

Author: update in [e5fdd2b](https://github.com/PrefectHQ/prefect/pull/11769/commits/e5fdd2bc9c96062aadea25e8c81df4948227afaa)

> - Fix environment variable not accepting lists — https://github.com/PrefectHQ/prefect/pull/11722

Author: ```suggestion
- Allow prefect settings to accept lists — https://github.com/PrefectHQ/prefect/pull/11722
```

- Revert deprecation of worker webserver setting — https://github.com/PrefectHQ/prefect/pull/11758

### Documentation

> - Detail `send_input` and `receive_input` — https://github.com/PrefectHQ/prefect/pull/11724

Reviewer: ```suggestion
- Expand docs on interactive flows, detailing `send_input` and `receive_input` — https://github.com/PrefectHQ/prefect/pull/11724
```

# Prefect Release Notes

## Release 2.14.18

### Fixes
- Fix environment variable not accepting lists — https://github.com/PrefectHQ/prefect/pull/11722
- Revert deprecation of worker webserver setting — https://github.com/PrefectHQ/prefect/pull/11758

### Documentation
- Detail `send_input` and `receive_input` — https://github.com/PrefectHQ/prefect/pull/11724

> 
> ### Uncategorized

Reviewer: ```suggestion
```

effectively moves this to docs section where it belongs

- Clarify that interval schedules use an anchor not start date — https://github.com/PrefectHQ/prefect/pull/11767

## New Contributors
* @clefelhocz2 made their first contribution in https://github.com/PrefectHQ/prefect/pull/11722

> ### Contributors
> - @abrookins
> - @clefelhocz2
> - @zhen0
> - @zzstoatzz

Reviewer: ```suggestion
```

> **All changes**: https://github.com/PrefectHQ/prefect/compare/2.14.17...preview

Reviewer: ```suggestion
**All changes**: https://github.com/PrefectHQ/prefect/compare/2.14.17...2.14.18
```
